cluster/kubernetes/kubernetes.go

```diff
@@ -240,7 +241,7 @@ func (c *Cluster) Export() ([]byte, error) {
 		}
 
 		for _, resourceKind := range resourceKinds {
-			workloads, err := resourceKind.getWorkloads(c, ns.Name)
+			workloads, err := resourceKind.getWorkloads(ctx, c, ns.Name)
```

Comment:
For this one I think you can get away with checking here, rather than in each `getWorkload` method. Similarly `getWorkloads`.

Reply:
That is what I first had, the reason I choose to not do this is that in case Flux gets extended in the future (i.e. some other action calling `getWorkload`), it is already covered.

Reply:
Fair enough, yup :+1: